upstart/cob-h2-start

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#################################################################
+##\file
+#
+# \note
+# Copyright (c) 2016 \n
+# Fraunhofer Institute for Manufacturing Engineering
+# and Automation (IPA) \n\n
+#
+#################################################################
+#
+# \note
+# Project name: care-o-bot
+# \note
+# ROS stack name: setup_cob4
+# \note
+# ROS package name: setup_cob4
+#
+# \author
+# Author: Nadia Hammoudeh Garcia, email:nadia.hammoudeh.garcia@ipa.fhg.de
+# \author
+# Supervised by: Nadia Hammoudeh Garcia, email:nadia.hammoudeh.garcia@ipa.fhg.de
+#
+# \date Date of creation: Feb 2016
+#
+#
+#################################################################
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# - Redistributions of source code must retain the above copyright
+# notice, this list of conditions and the following disclaimer. \n
+# - Redistributions in binary form must reproduce the above copyright
+# notice, this list of conditions and the following disclaimer in the
+# documentation and/or other materials provided with the distribution. \n
+# - Neither the name of the Fraunhofer Institute for Manufacturing
+# Engineering and Automation (IPA) nor the names of its
+# contributors may be used to endorse or promote products derived from
+# this software without specific prior written permission. \n
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License LGPL as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License LGPL for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License LGPL along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+#################################################################
+
+export ROS_MASTER_URI=http://myrobot-b1:11311
```

Comment:
yes, this script has to be still tested on the robot, if it works I will create an install script where the robot name will be automatically set.
For example: https://github.com/ipa320/setup_cob4/blob/master/upstart/upstart_install.sh#L11